com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/EvaluateTipThicknessWaterNode.cs

```diff
@@ -0,0 +1,49 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class EvaluateTipThicknessWaterNode : IStandardNode
```

Comment:
You should rename this one to EvaluateWaterTipThickness to keep your renaming consistent with the others.

Reply:
good catch!